test/CMakeLists.txt

```diff
@@ -115,9 +115,9 @@ endif()
 if(LINUX)
     # the base_alloc test uses linux pthreads
     add_umf_test(NAME base_alloc
-                SRCS test_base_alloc.c
+                SRCS ${BA_SOURCES} test_base_alloc.c
```

Comment:
I don't get it - you add these files both to libumf and tests?

Reply:
Yes, as I wrote in the commit description, BA was being compiled as part of libumf (not a separate library) but BA symbols should not be visible outside of the libumf.so/.a/.dll. On windows, since those symbols were not exported, the linking of this test would fail (same for linux after we introduce linker map). So, the solution is to either use those sources directly in tests (that's what I did) or make BA a static library (but I don't see a need for that right now).